### PR TITLE
feat(metrics): integrate Glean into content server backend

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - the content server backend needs Glean to record some RP related events at a minimum

This commit
 - adds, configures, and initializes Glean for the content server backend

Also note that the server_events.js in this patch was generated with a patched version of glean_parser (https://github.com/mozilla/glean_parser/pull/726) so if you re-generate with `yarn glean-generate`, you'd end up with ESM and things will break.

### CODE REVIEW

File: packages/fxa-content-server/server/lib/configuration.js [hunk 1]
```diff
@@ -332,6 +332,32 @@ const conf = (module.exports = convict({
     env: 'FXA_URL',
     format: 'url',
   },
+  serverGleanMetrics: {
+    enabled: {
+      default: false,
```
Comment (chenba) [inline packages/fxa-content-server/server/lib/configuration.js:337]: Keeping this to false until the code actually populate the metric values.

**Comment by vpomerleau** [comment]
What's the alternative to running `yarn glean-generate`, if it will break things?

**Comment by chenba** [comment]
> What's the alternative to running `yarn glean-generate`, if it will break things?

That's temporary.  I don't think this needs to be blocked by a glean.js release.  If you want to try generating that file, the CommonJS patch has landed in the main branch of glean_parser.

Actually this reminded me that I need to update `glean-generate` in package.json.  Thanks!

File: packages/fxa-content-server/server/lib/glean/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  gleanServerEventLogger = serverGleanEvents.createEventsServerEventLogger({
+    applicationId: config.serverGleanMetrics.applicationId,
+    appDisplayVersion: config.version,
+    channel: config.serverGleanMetrics.channel,
+    logger_options: { app: config.serverGleanMetrics.loggerAppName },
+  });
+
+  return {
+    rp: {
+      formView: async (metrics, rpFormType) => {
```
Comment (chenba) [inline packages/fxa-content-server/server/lib/glean/index.js:29]: I added an event so I can verify that something is showing up in the log.  The impl here and the call site will be updated in a follow-up patch.

**Review by vbudhram — APPROVED** [review]
@chenba Code looks good 👍🏽

**Pull request merged**